cypress/integration/seededFlows/adminFlows/users/manageCredits.spec.js

```diff
@@ -0,0 +1,83 @@
+// More on roles, https://admin.forem.com/docs/forem-basics/user-roles
+function openCreditsModal() {
+  cy.getModal().should('not.exist');
+  cy.findByRole('button', { name: 'Adjust balance' }).click();
+
+  return cy.getModal();
+}
+
+// function closeUserUpdatedMessage() {
+//   cy.findByText('User has been updated').should('exist');
+//   cy.findByRole('button', { name: 'Close' }).click();
+//   cy.findByText('User has been updated').should('not.exist');
+// }
+
+describe('Manage User Credits', () => {
+  describe('As an admin', () => {
+    beforeEach(() => {
+      10;
+      cy.testSetup();
+      cy.fixture('users/adminUser.json').as('user');
+      cy.get('@user').then((user) => {
+        cy.loginAndVisit(user, '/admin/users/8');
+      });
+    });
+
+    it('should add credits', () => {
+      cy.findByTestId('user-credits').should('have.text', '100');
+
+      openCreditsModal().within(() => {
+        cy.findByRole('combobox', { name: 'Adjust balance' }).select('Add');
+        cy.findByRole('spinbutton', {
+          name: 'Amount of credits to add or remove',
+        }).type('10');
+        cy.findByRole('textbox', {
+          name: 'Why are you adjusting credits?',
+        }).type('some reason');
+        cy.findByRole('button', { name: 'Adjust' }).click();
+      });
+
+      cy.getModal().should('not.exist');
+      // closeUserUpdatedMessage(); // TODO
```

Comment:
Still need to sort out the TODOs.

Reply:
I remember why these are here now. Currently, there are no flash messages when updating credits.

Reply:
@JennieOcken, @Ridhwana do we have any particular wording we want for notifying the admin that credits were added or removed? If not, I'm happy to write something up and wire it up with a flash message.

Reply:
cc'ing @amywtlin here too.

Reply:
Hey @nickytonline - I'll work with updating the overall copy with @juliannatetreault together later on. Please feel free to move forward with all the existing copies (within the member detail view project) and we'll take care of them afterwards.